webpack.config.js

```diff
@@ -43,6 +43,31 @@ module.exports = (env = {}) => {
         info.resourcePath.replace(/^\.\//, ''),
       crossOriginLoading: 'anonymous',
     },
+    optimization: {
+      splitChunks: {
+        cacheGroups: {
+          commons: {
+            name: 'vendor',
+            test: /[\\/]node_modules[\\/]/,
+            chunks: function(module) {
```

Comment:
This doesn't seem to be splitting chunks right. Here's the current output of running `yarn build:analyze` on this branch:
<img width="1919" alt="Screen Shot 2019-06-26 at 10 28 18 AM" src="https://user-images.githubusercontent.com/756501/60201522-4f08af00-97fd-11e9-88c4-8d76addc2f03.png">

I changed the config to match https://github.com/CruGlobal/missionhub-web/blob/master/webpack.config.js#L48-L58 and got this:
![Screen Shot 2019-06-26 at 10 28 25 AM](https://user-images.githubusercontent.com/756501/60201523-4f08af00-97fd-11e9-82bf-5a3e1bdf11f1.png)

I couldn't instantly find good docs on what `chunks: 'initial',` does and don't remember why I did that before... Webpack is always fun :)

Reply:
Should I leave the commons2, manifest, to have a minChunks of Infinity still? If not I'll push up the change with the new config of initial :slightly_smiling_face: I did not know yarn build:analyze was a thing but that is super cool!

Reply:
I don't really know what that does. If there's a reason to do it that should be good. But simpler config is always nice :)

Reply:
Yeah I'm not sure either. Simple is definitely better. Webpack has been a learning experience for sure :joy: